_field-types/supported-field-types/match-only-text.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Match only text
+nav_order: 61
+has_children: false
+parent: String field types
+grand_parent: Supported field types
+redirect_from:
+  - /opensearch/supported-field-types/match-only-text/
+  - /field-types/match-only-text/
+---
+
+# Match only text field type
+
+A variant of text field designed for full-text search when scoring and positional information of terms within a document are not critical.
+
+Key features:
+ - Omits positions, frequencies, and norms reducing storage.
+ - Disables scoring, resulting in all matching documents receiving a constant score of 1.0.
+ - Supports most query types, but not interval/span queries.
+
+
+Choosing `match_only_text` means prioritizing efficient full-text search over complex ranking and positional queries, while also optimizing storage costs. It excels when you need to quickly find documents containing specific terms without the overhead of storing frequencies and positions, leading to significantly smaller indexes. This translates to lower storage costs, especially for large datasets. However, it's not the best choice for ranking results based on relevance or for queries that hinge on term proximity or order, like interval/span queries. While it does support phrase queries, their performance isn't as efficient as with text field type. So, if pinpointing exact phrases or their locations within documents is essential, consider using text field type.
```

Comment:
> While it does support phrase queries, their performance isn't as efficient as with text field type. So, if pinpointing exact phrases or their locations within documents is essential, consider using text field type.

Does this maybe downplay it too much? `match_only_text` supports phrase queries but performance will likely be very very bad, right? I'd maybe tweak the last sentence:

"So, if pinpointing exact phrases or their locations within documents is essential, use the `text` field type instead."

Reply:
Makes sense! In addition to what you're suggesting, maybe I can add a line on how phrase queries work here and how bad/optimal they could depending on query and workload.

Reply:
done